tracing-core/Cargo.toml

```diff
@@ -25,8 +25,10 @@ categories = [
 ]
 keywords = ["logging", "tracing"]
 edition = "2018"
+
 [badges]
 travis-ci = { repository = "tokio-rs/tracing", branch = "master" }
```

Comment:
can we drop this?

Reply:
We can, but since we've not disabled travis, I figured I'd leave it? I could go either way.

Reply:
I say we drop it since really all that matters is azure right?

Reply:
i went ahead and removed it